config/application.rb

```diff
@@ -69,5 +69,9 @@ class Application < Rails::Application
     config.force_jwt_at_registration = true
 
     config.feature_flag_mfa = ENV["FEATURE_FLAG_MFA"] == "enabled"
+
+    config.i18n.load_path += Dir[Rails.root.join("config/locales/**/*.yml").to_s]
+
```

Comment:
Looking again at the Rails intl guide there's two others config settings we might want to consider whilst we're here:

```
# Permitted locales available for the application
I18n.available_locales = [:en]
```

And perhaps

```
I18n.enforce_available_locales
```

I'd want to check but I'd suggest that would scream and shout at us if we tried (or accidentially) added other locales without being concious / deliberate about it?

I think we should raise actual other languages with the team and see if we can add some more widely spoken options to anyone who requests it, so could come in useful?

What do others think?

Reply:
I'm not really fussed either way.  If we add a translation for another language that isn't permitted, then I'd hope we realise when the tests fail and manual testing reveal the translation isn't working correctly.

Reply:
From [the rails docs](https://guides.rubyonrails.org/configuring.html#configuring-i18n), `I18n.enforce_available_locales` defaults to true.

> I'd want to check but I'd suggest that would scream and shout at us if we tried (or accidentially) added other locales without being concious / deliberate about it?

It sounds like what it'd do is throw an error if we try to explicitly use, in the code, a locale which isn't in the `available_locales` list; rather than pro-actively throwing an error if such a locale file is present.

Reply:
ah great to know about the defaults.
cool cool, so sounds like consensus is to leave `available_locales` aside.
Will resolve this thread